packages/js/src/ui/components/Notification/NotificationList.tsx

```diff
@@ -47,24 +35,31 @@ type NotificationListProps = {
   filter?: NotificationFilter;
 };
 
-/* This is also going to be exported as a separate component. Keep it pure. */
 export const NotificationList = (props: NotificationListProps) => {
   const options = createMemo(() => ({ ...props.filter, limit: props.limit }));
+  const style = useStyle();
   const { data, setEl, end, refetch, initialLoading } = useNotificationsInfiniteScroll({ options });
   const { count, reset: resetNewMessagesCount } = useNewMessagesCount({ filter: { tags: props.filter?.tags ?? [] } });
+  let notificationListElement: HTMLDivElement;
 
   const handleOnNewMessagesClick: JSX.EventHandlerUnion<HTMLButtonElement, MouseEvent> = async (e) => {
     e.stopPropagation();
     resetNewMessagesCount();
     refetch({ filter: props.filter });
+    notificationListElement.scrollTo({ top: 0 });
   };
 
   return (
-    <>
+    <div class={style('notificationListContainer', 'nt-h-full nt-overflow-hidden')}>
```

Comment:
Add this because it makes sense for the structure and we will need it here if we ever export just a list component for the notifications.

Reply:
`nt-overflow-hidden` is needed for the list below to be scrollable. It sadly clips the CTA animation (because it comes from the top`. Maybe we can make that just fade in.